src/dmn/trn_transit_xdp_usr.c

```diff
@@ -304,6 +318,18 @@ int trn_get_endpoint(struct user_metadata_t *md, struct endpoint_key_t *epkey,
 	return 0;
 }
 
+int trn_get_packet_metadata(struct user_metadata_t *md, struct packet_metadata_key_t *key,
+		     struct packet_metadata_t *packet_metadata)
+{
+	int err = bpf_map_lookup_elem(md->packet_metadata_map_fd, key, packet_metadata);
+	if (err) {
+		TRN_LOG_ERROR("Querying packet metadata mapping failed (err:%d).",
+			      err);
```

Comment:
nit: do we really need err to be on the next line?

Reply:
Fixed.